test/unit/webdriver/device/keyboard_test.py

```diff
@@ -32,3 +32,27 @@ def test_hide_keyboard(self):
             appium_command('/session/1234567890/appium/device/hide_keyboard')
         )
         assert isinstance(driver.hide_keyboard(), WebDriver)
+
```

Comment:
Please add test cases for other Keyboard Apis, not only below 2 apis.

Reply:
I saw somewhere KeyEvent is Selendroid - So didn't add it!
What else?

Reply:
You can see other 3 apis in below file.
https://github.com/appium/python-client/blob/9e18dfce1ccbe34a5bba3684b054e7f5e22bd229/appium/webdriver/extensions/keyboard.py

```
is_keyboard_shown
hide_keyboard
keyevent
```

Reply:
will add tests for metastate and flag args